android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/WebBrowserActivity.java

```diff
@@ -381,13 +377,8 @@ public void onBackPressed() {
     if (webView != null && webView.canGoBack()) {
       webView.goBack();
     } else {
-      // Hide the current keyboard so when Keyman app returns, there aren't 2 keyboards visible
-      InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
-      imm.hideSoftInputFromWindow(addressField.getWindowToken(), 0);
-
       super.onBackPressed();
```

Comment:
All these workarounds can be removed now that the `MainActivity.onResume()` handles hiding the system OSK